packages/core/src/routes/sign-in-experience.test.ts

```diff
@@ -30,28 +43,23 @@ describe('signInExperiences routes', () => {
   });
 
   it('PATCH /sign-in-exp', async () => {
-    const branding: Branding = {
-      primaryColor: '#000',
-      backgroundColor: '#fff',
-      darkMode: true,
-      darkBackgroundColor: '#000',
-      darkPrimaryColor: '#fff',
-      style: BrandingStyle.Logo,
-      logoUrl: 'http://silverhand.png',
-      slogan: 'silverhand',
-    };
-
+    const termsOfUse: TermsOfUse = { enabled: false };
     const socialSignInConnectorIds = ['abc', 'def'];
 
     const response = await signInExperienceRequester.patch('/sign-in-exp').send({
-      branding,
+      branding: mockBranding,
+      termsOfUse,
       socialSignInConnectorIds,
     });
 
+    expect(validateBranding).toHaveBeenCalledWith(mockBranding);
+    expect(validateTermsOfUse).toHaveBeenCalledWith(termsOfUse);
+
     expect(response.status).toEqual(200);
     expect(response.body).toEqual({
```

Comment:
nit: can use .toMatchObject() and avoid completing the result.

Reply:
Updated. PTAL